front/app/utils/auth/jwt.ts

```diff
@@ -2,6 +2,8 @@ import { get, set, remove } from 'js-cookie';
 import jwtDecode from 'jwt-decode';
 
 const COOKIE_NAME = 'cl2_jwt';
+const SECUIRE_COOKIE =
+  process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'test';
```

Comment:
Could you explain this one? 

MDN:
> Indicates that the cookie is sent to the server only when a request is made with the https: scheme (except on localhost), and therefore, is more resistant to man-in-the-middle attacks.

For dev environment, it wouldn't work in any case (localhost) or am I misunderstanding MDN?

Reply:
There might be 2 cases:
1. If you want to test tenant fetching, you might want to access the app from a local domain like `test-citizenlab.com`. It's still development env, but not localhost.
2. In e2e tests we access the app using `http://e2e.front` domain. Again, it's not localhost.